src/ui/Questionnaire/Questionnaires.jsx

```diff
@@ -19,24 +30,123 @@ export function Questionnaires({ surveyUnit }) {
   return (
     <Card p={2} elevation={0}>
       <CardContent>
-        <Stack gap={3}>
-          <Row gap={1}>
-            <StickyNote2Icon fontSize="large" />
-            <Typography as="h2" variant="xl" fontWeight={700}>
-              {D.openQuestionnaire}
-            </Typography>
-          </Row>
-          <Button
-            variant="contained"
-            disabled={!isAvailable}
-            startIcon={<LibraryBooksIcon />}
-            component={RouterLink}
-            to={`/queen/survey-unit/${id}`}
-          >
-            {D.accessTheQuestionnaire}
-          </Button>
+        <Stack gap={6}>
+          <Stack gap={3}>
+            {/* Title */}
+            <Row gap={4}>
+              <Row gap={1}>
+                <StickyNote2Icon fontSize="large" />
+                <Typography as="h2" variant="xl" fontWeight={700}>
+                  {D.openQuestionnaire}
+                </Typography>
+              </Row>
+
+              {isAvailable && (
+                <Chip
+                  label={D.inProgress}
+                  icon={<TrackChangesIcon />}
+                  size="small"
+                  color="warning"
+                  sx={{ ...chipStyle, color: 'warning.main' }}
+                />
+              )}
+
+              <Button
+                variant="contained"
+                disabled={!isAvailable}
+                startIcon={<SlowMotionVideoIcon />}
+                component={RouterLink}
+                to={`/queen/survey-unit/${id}`}
+              >
+                {D.accessTheQuestionnaire}
+              </Button>
+            </Row>
+          </Stack>
+          <Stack gap={3}>
+            {/* Title */}
+            <Row justifyContent="space-between">
+              <Row gap={1}>
+                <GroupOutlinedIcon fontSize="large" />
+                <Typography as="h2" variant="xl" fontWeight={700}>
+                  {D.personDetails}
+                </Typography>
+              </Row>
+            </Row>
+
+            {/* Table */}
+            <ArticulationTable id={id} />
```

Comment:
Can you had a feature flag with env variable ?

`VITE_ARTICULATION_ENABLED=true/false` (false by default)

In order to activate/desactivate this feature

```ts
{ import.meta.env.VITE_ARTICULATION_ENABLED === 'true' && <ArticulationTable id={id}/> }
```